.gitignore

```diff
@@ -11,6 +11,9 @@ neurovault/static/
 
 # test data
 neurovault/apps/statmaps/tests/test_data/feat/*
+neurovault/apps/statmaps/tests/bench/*
+neurovault/apps/statmaps/templates/statmaps/statisticmap_details.html.html
```

Comment:
Yes, sorry for that... in bench/  I'm saving the test data. And the statisticmap_details file is a bug that I reported as Issue. Pycharm keeps generating that file, so I encountered no other way of git telling me about that new file. I will delete both lines.